api/v1alpha1/perses_types.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2023 The Perses Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type FileExtension string
```

Comment:
Let's have this annotation and we can limit the values on the CRD.
`+kubebuilder:validation:Enum=JSON, YAML`

Reply:
maybe more `+kubebuilder:validation:Enum=json,yaml`

Reply:
done